csharp/ql/src/semmle/code/csharp/dataflow/internal/DataFlowPrivate.qll

```diff
@@ -2354,7 +2354,7 @@ private predicate viableConstantBooleanParamArg(
   )
 }
 
-int accessPathLimit() { result = 3 }
+int accessPathLimit() { result = 5 }
```

Comment:
I expected to see some performance change, but it looks like there's basically nothing. Why?

Reply:
When pruning works sufficiently well, the overall performance of the data flow library is something like "reachable PathNodes" + "size of the first pruning stage" (I claim, based on handwaving, intuition, and experience). So if there isn't a lot of new flow paths that use accesspaths of length 4 and 5 (which is very plausible), then performance might very well be unchanged. Chalk it up to the awesomeness of great pruning!